hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/cache/CacheRemovePartitionLostListenerMessageTask.java

```diff
@@ -38,8 +37,8 @@ public CacheRemovePartitionLostListenerMessageTask(ClientMessage clientMessage,
     @Override
     protected Object call() {
         ICacheService service = getService(CacheService.SERVICE_NAME);
```

Comment:
Good catch. I have updated most of them as `ICacheService.SERVICE_NAME` but I had missed a few of them :)